modules/bridge/src/main/ts/ephox/bridge/components/dialog/Button.ts

```diff
@@ -25,6 +26,7 @@ export interface Button {
   icon: Optional<string>;
   borderless: boolean;
   buttonType: Optional<'primary' | 'secondary' | 'toolbar'>;
+  showIconAndText?: boolean;
```

Comment:
We don't need this one anymore right? Since view can do this:
```
    {
      name: 'icon',
      type: 'togglableIconButton',
      tooltip: 'togglableIcon',
      icon: 'info',
      onAction: (statusApi) => {
      }
    },
    {
      name: 'iconAndText',
      type: 'togglableIconButton',
      text: 'iconAndText',
      tooltip: 'iconAndText',
      icon: 'copy',
      borderless: true,
      onAction: (api) => {
      }
    }
```
And dialogs can do this:
```
      {
        type: 'togglableIconButton',
        name: 'icon',
        icon: 'bold',
        tooltip: 'Tooltip'
      },
      {
        type: 'togglableIconButton',
        name: 'iconAndText',
        text: 'Bold',
        icon: 'bold'
      },

```

Reply:
removed

Reply:
Please update the PR description as well, I think it's now inaccurate

Reply:
updated